sasl/plain/src/main/java/org/wildfly/security/sasl/plain/PlainSaslClient.java

```diff
@@ -92,7 +94,7 @@ public boolean hasInitialResponse() {
             b.append((byte) 0);
             StringPrep.encode(name, b, StringPrep.PROFILE_SASL_STORED);
```

Comment:
For consistency, I think it makes sense to update the other two usages of ```StringPrep.PROFILE_SASL_STORED``` to ```this.passwordProfile``` and then also update the authentication id used in the test case accordingly.

Reply:
done